packages/smart/tasks/fundLink.ts

```diff
@@ -0,0 +1,22 @@
+import { task, types } from "hardhat/config";
+
+import "hardhat/types/config";
+import { makeSigner } from "./deploy";
+import { LinkTokenInterface__factory } from "../typechain";
+
+task("fundLink", "Send 1 link to a contract on kovan")
+  .addParam("contract", undefined, undefined, types.string)
+  .setAction(async ({ contract: contractAddress }, hre) => {
+    if (typeof contractAddress !== "string") return;
+    const { ethers } = hre;
+    const signer = await makeSigner(hre);
+
+    const linkTokenAddress = "0xa36085F69e2889c224210F603D836748e7dC0088";
+    const amount = ethers.BigNumber.from(10).pow(18);
+    const linkToken = LinkTokenInterface__factory.connect(linkTokenAddress, signer);
+
+    await linkToken.transfer(contractAddress, amount).then((tx: any) => {
```

Comment:
You shouldn't need to specify the type of `tx`. We're using TypeChain now, which adds type info to contract calls.